x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/notes_tab.cy.ts

```diff
@@ -32,7 +32,7 @@ const link = 'https://www.elastic.co/';
 describe('Timeline notes tab', { tags: ['@ess', '@serverless'] }, () => {
   beforeEach(function () {
     deleteTimelines();
-
+    login();
```

Comment:
how was this ticket even working before????

Reply:
It was a merge error with @MadameSheema 's changes. She moved the login to line 37 here :) . Thanks!

Reply:
then shouldn't we remove it from line 37?